galeracluster/source/glbparameters.rst

```diff
@@ -0,0 +1,705 @@
++----------------------------+--------------------------------+
+
+This is an optional parameter.  It defines the path to the FIFO control file as is always opened.  In the event that there is already a file at this path, Galera Load Balancer fails to start.
+
+.. code-block:: ini
+
+   CONTROL_FIFO="/var/run/glbd.fifo"
+
+   
+.. rubric:: ``DEFAULT_TARGETS``
+.. _`glb-default_targets`:
+
+Defines the IP addresses and ports of the destination servers.
+
++----------------------------+--------------------------------+
+| **Default Configuration**  | ``127.0.0.1:80 10.0.0.1:80     |
+|                            | 10.0.0.2:80:2``                |
++----------------------------+--------------------------------+
+| **Mandatory Parameter**    | No                             |
++----------------------------+--------------------------------+
+
+This parameter defines that IP addresses that Galera Load Balancer uses as destination servers.  Specifically, in this case the Galera Cluster nodes that it routes application traffic onto.
+
+.. code-block:: ini
+
+   DEFAULT_TARGETS="192.168.1.1 192.168.1.2 192.168.1.3"
+   
+
+.. rubric:: ``LISTEN_ADDR``
+.. _`glb-listen_addr`:
+
+Defines the IP address and port used for client connections.
+
++----------------------------+--------------------------------+
+| **Default Configuration**  | ``8010``                       |
++----------------------------+--------------------------------+
+| **Mandatory Parameter**    | Yes                            |
++----------------------------+--------------------------------+
+
+This parameter defines the IP address and port that Galera Load Balancer listens on for incoming client connections.  The IP address is optional, the port mandatory.  In the event that you define a port without an IP address, Galera Load Balancer listens on that port for all available network interfaces.
+
+.. code-block:: ini
+
+   LISTEN_ADDR="8010"
+
+
+
+.. rubric:: ``MAX_CONN``
+.. _`glb-max_conn-par`:
+
+Defines the maximum allowed client connections.
+
++----------------------------+--------------------------------+
+| **Command-line Argument**  | :ref:`--max_conn               |
+|                            | <glb-max_conn-arg>`            |
++----------------------------+--------------------------------+
+| **Mandatory Parameter**    | No                             |
++----------------------------+--------------------------------+
+
+This parameter defines the maximum number of client connections that you want to allow to Galera Load Balancer.  It modifies the system open files limit to accommodate at least this many connections, provided sufficient privileges.  It is recommend that you define this parameter if you expect the number of client connections to exceed five hundred.
+
+.. code-block:: ini
+
+   MAX_CONN="135"
+
+
+This option defines the maximum number of client connections that you want allow to Galera Arbitrator. Bear in mind, that it can be operating system dependent.
```

Comment:
I think this sentence is redundant. Also, not sure how the Arbitrator got mentioned here.